lib/solvers/AutoroutingPipelineSolver.ts

```diff
@@ -368,13 +370,23 @@ export class AutoroutingPipelineSolver extends BaseSolver {
         },
       ],
     ),
+    definePipelineStep("sameNetViaMerger", SameNetViaMergerSolver, (cms) => [
+      {
+        unsimplifiedHdRoutes:
```

Comment:
extremely confusing name because these routes appear POST-simplification

```suggestion
        inputHdRoutes:
```